scripts/mockapi.js

```diff
@@ -0,0 +1,66 @@
+const marginActions = {
+  sell: 'https://secure.icicidirect.com/trading/equity/marginsell',
+  buy: 'https://secure.icicidirect.com/trading/equity/marginbuy',
+};
+
+const mockPredicationConstant = {
+  recoPrice: 'Reco. Price',
+  cmp: 'CMP',
+  targetPrice: 'Target Price',
+  stopLoss: 'Stop Loss',
+  profitPotential: 'Profit Potential',
+  returns: 'Returns',
+  action: 'Action',
+  profitExit: 'Call Closed and Book Profit Price:',
+  minAmount: 'Min. Amount',
+  riskProfile: 'Risk Profile',
+  buyingRange: 'Buying Range',
+
+};
+
+const apiEndPoints = {
+  trading: '/draft/anagarwa/tradingtips.json',
+  investing: '/draft/anagarwa/investingideas.json',
+  oneclickportfolio: '/draft/anagarwa/oneclickportfolio.json',
+  muhratpicks: '/draft/anagarwa/muhratpicks.json',
+};
+
+async function fetchRecommendations(type) {
+  const apiUrl = `${window.location.origin}${apiEndPoints[type]}`;
+  try {
+    const response = await fetch(apiUrl);
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`);
+    }
+    const data = await response.json();
+    // Transform the API response to the desired companies array format
+    const companies = data.data.map((company) => ({
+      action: company.action,
```

Comment:
Seems the company name is clickable, do we fetch the data from api call?
<img width="290" alt="image" src="https://github.com/aemsites/icicidirect/assets/57093988/a30633e5-95af-4545-a17c-d1bd742b192f">

Reply:
@JiangLong2019  yes these data will be fetched from API. Right now I have created  a mock API,
And in original website this is not a clickable link.